handlers/metrics_test.go

```diff
@@ -57,51 +131,36 @@ func TestBountyMetrics(t *testing.T) {
 	})
 
 	t.Run("should fetch stats from db", func(t *testing.T) {
-		db.RedisError = errors.New("redis not initialized")
 		rr := httptest.NewRecorder()
 		handler := http.HandlerFunc(mh.BountyMetrics)
-		workspace := "test-workspace"
 
-		dateRange := db.PaymentDateRange{
-			StartDate: "1111",
-			EndDate:   "2222",
-		}
 		body, _ := json.Marshal(dateRange)
-		req, err := http.NewRequestWithContext(ctx, http.MethodPost, "/bounty_stats?workspace="+workspace, bytes.NewReader(body))
+		req, err := http.NewRequestWithContext(ctx, http.MethodPost, "/bounty_stats?workspace=workspace", bytes.NewReader(body))
 		if err != nil {
 			t.Fatal(err)
 		}
-		mockDb.On("TotalBountiesPosted", dateRange, workspace).Return(int64(1)).Once()
-		mockDb.On("TotalPaidBounties", dateRange, workspace).Return(int64(1)).Once()
-		mockDb.On("TotalAssignedBounties", dateRange, workspace).Return(int64(2)).Once()
-		mockDb.On("BountiesPaidPercentage", dateRange, workspace).Return(uint(1)).Once()
-		mockDb.On("TotalSatsPosted", dateRange, workspace).Return(uint(1)).Once()
-		mockDb.On("TotalSatsPaid", dateRange, workspace).Return(uint(1)).Once()
-		mockDb.On("SatsPaidPercentage", dateRange, workspace).Return(uint(1)).Once()
-		mockDb.On("AveragePaidTime", dateRange, workspace).Return(uint(1)).Once()
-		mockDb.On("AverageCompletedTime", dateRange, workspace).Return(uint(1)).Once()
-		mockDb.On("TotalHuntersPaid", dateRange, workspace).Return(int64(1)).Once()
-		mockDb.On("NewHuntersPaid", dateRange, workspace).Return(int64(1)).Once()
+
 		handler.ServeHTTP(rr, req)
+		assert.Equal(t, http.StatusOK, rr.Code)
 
 		expectedMetricRes := db.BountyMetrics{
-			BountiesPosted:         1,
-			BountiesPaid:           1,
-			BountiesAssigned:       2,
-			BountiesPaidPercentage: 1,
-			SatsPosted:             1,
-			SatsPaid:               1,
-			SatsPaidPercentage:     1,
-			AveragePaid:            1,
-			AverageCompleted:       1,
-			UniqueHuntersPaid:      1,
-			NewHuntersPaid:         1,
+			BountiesPosted:         4,
```

Comment:
@aliraza556 SatsPaid, SatsPosted, BountiesPaid, BountieAssigned should not be a static variable

Reply:
@elraphty Ok